config/config.yaml

```diff
@@ -30,12 +30,13 @@ service:
     SECRET_KEY: aardvark
     EMAIL_TOKEN_SALT: aardvark
     PARTY_SCHEMA: ras_party/schemas/party_schema.json
+    PUBLIC_EMAIL_VERIFICATION_URL: http://ons.survey.gov/bres                   # Do not add trailing slash e.g. http://x.y/parent/child/ BAD http://x.y/parent/child GOOD
 
 dependencies:
     ras-party-db:
         schema: partysvc
-        #uri: "sqlite:///ras-party"
-        uri: "postgresql://postgres:postgres@localhost:5432/postgres"
+        uri: "sqlite:///ras-party"
+        #uri: "postgresql://postgres:postgres@localhost:5432/postgres"
```

Comment:
Not sure why this comment is here

Reply:
it's a way to get the system to run locally. reversed them, so back to normal